src/pages/patientView/PatientViewPageTabs.tsx

```diff
@@ -175,7 +177,10 @@ export function tabs(
                     pageComponent.patientViewPageStore.cnaSegments.result
                         .length > 0) &&
                 sampleManager && (
-                    <div>
+                    <FeatureInstruction
+                        content={'Click location for detail'}
```

Comment:
Is it clear what `location` means? Should we say `location on a feature track`? `Feature track` may also be confusing though. Maybe location is okay as is.